scripts/integration/webhdfs/test.yaml

```diff
@@ -0,0 +1,11 @@
+test_filter: '::webhdfs::'
+
+features:
+- webhdfs-integration-tests
+
+env:
+  WEBHDFS_ENDPOINT: http://namenode.local:9870
+  LD_LIBRARY_PATH: $LD_LIBRARY_PATH:/usr/lib/jvm/default/lib/server
```

Comment:
This isn't needed is it, just a carry over from `hdfs`?

Reply:
Yes, we don't need it.